dist/react-datepicker.js

```diff
@@ -440,6 +440,15 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	    var timeFormats = ["HH:mm:ss a"];
 
+	    var manualDateString = this.state.manualDate;
+
+	    // Add a space between a number and pm if needed
+	    if (manualDateString) {
+	      for (var i = 0; i <= 9; i++) {
+	        manualDateString = manualDateString.replace(new RegExp(i.toString() + "pm", "g"), i.toString() + " pm");
+	      }
```

Comment:
Elaborating: you use parentheses `(some match)` (sometimes you have to escape the parens) to capture a match, and then typically insert the match in the replacement using `\1` or `$1`